hera_mc/cm_hookup.py

```diff
@@ -419,14 +424,18 @@ def add_correlator_levels(self, hookup_dict):
                 level_ctr += 1
         return hookup_dict
 
-    def __write_hookup_cache_to_file(self):
+    def __write_hookup_cache_to_file(self, force_new):
         with open(self.hookup_cache_file, 'wb') as f:
             np.save(f, self.at_date)
             np.save(f, cm_utils.stringify(self.hookup_list_to_cache))
             np.save(f, self.cached_hookup_dict)
             np.save(f, self.part_type_cache)
+        log_dict = {'hu-list': cm_utils.stringify(self.hookup_list_to_cache),
+                    'check_hookup': str(self.__check_hookup_cache_file_date()),
+                    'force_new': str(force_new)}
+        cm_utils.log('update_cache', log_dict=log_dict)
```

Comment:
If I'm reading this correctly, the only thing this function does with the `force_new` argument is log it. The function shouldn't require arguments that it doesn't really use. With the right logging info we can probably infer the value of `force_new` anyway.

Reply:
Yes - and as soon as we figure out what is going on I'll remove this.  This was just an expedient way to do it for now.